drivers/video/video_ctrls.c

```diff
@@ -592,13 +593,12 @@ int video_query_ctrl(const struct device *dev, struct video_ctrl_query *cq)
 	return 0;
 }
 
-void video_print_ctrl(const struct device *const dev, const struct video_ctrl_query *const cq)
+void video_print_ctrl(const struct video_ctrl_query *const cq)
 {
 	uint8_t i = 0;
 	const char *type = NULL;
 	char typebuf[8];
 
-	__ASSERT_NO_MSG(dev != NULL);
```

Comment:
[non-blocking]
If wanting to preserve the assert, this could become:
```suggestion
	__ASSERT_NO_MSG(cq->dev != NULL);
```